### ISSUE TITLE
Workers Never Register

### ISSUE THREAD

**Issue by avanier** [avanier/knc_concourse#4]
Currently, nothing works.

`web` and `worker` start with no errors, but `worker` never connects to `tsa`. No heartbeats are ever sent. Just total radio silence.

`strace` -ing `worker` shows :

```plaintext
strace: Process 24814 attached
futex(0x131a40f0, FUTEX_WAIT, 0, NULL
```

```shell
$ fly login -t vagrant -c http://127.0.0.1:8080
$ fly -t vagrant workers
name  containers  platform  tags  team  state  version
$ echo -e "\ue413"
😭
```

**Comment by avanier** [comment]
Halp! @typositoire

**Comment by avanier** [comment]
Was related to [concourse/1390](https://github.com/concourse/concourse/issues/1390). Fixed in `1.0.2`.
